api-server/migrations/20210828215333-refactor-date-storage.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.query(`
+      CREATE VIEW public.datetime_venue AS
+      SELECT public.uuid_generate_v4() as id,
+        id AS event_id,
+        venue_id,
+        start_time,
+        end_time,
+        optional_title
```

Comment:
I don't think we need to spell out that it's optional in the name, that should be made clear by the type system/api swagger and such. When we define this as a table I image it will be something like

```
      CREATE TABLE if NOT EXISTS datetime_venue(
        event_id VARCHAR(255) NOT NULL
            CONSTRAINT datetime_venue_event_id REFERENCES events,
        venue_id VARCHAR(255) NOT NULL
           CONSTRAINT datetime_venue_venue_id REFERENCES venues,
       start_time timestamp with time zone NOT NULL,
       end_time timestamp with time zone NOT NULL,
       title VARCHAR(255)
      );
```